src/hooks/useDeposit.ts

```
@@ -0,0 +1,160 @@
+      return { deposits: validDeposits, _credentialType, _totalDepositAmountBN };
+    },
+    [contractConfig, client]
+  );
+
+  const setDepositData = useCallback(
+    async (file: File) => {
+      if (file) {
+        let data: DepositDataJson[] = [];
+        try {
+          data = JSON.parse(await file.text());
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)